addons/common/functions/fnc_showHud.sqf

```diff
@@ -0,0 +1,58 @@
+/*
+ * Author: PabstMirror
+ * Allows multiple sources to not overwrite showHud command
+ * Bitwise AND Logic (a single false in a mask will make it false)
+ *
+ * Arguments:
+ * 0: Source ID <STRING><OPTIONAL>
+ * 1: Show Hud Bool Array (8 to set, empty to remove) <ARRAY><OPTIONAL>
```

Comment:
Parameters:
\[hud, info, radar, compass, direction, menu, group, cursors\]: Array

hud: Boolean - show scripted HUD (same as normal showHUD true/false)
info: Boolean - show vehicle + soldier info (hides weapon info from the HUD as well)
radar: Boolean - show vehicle radar
compass: Boolean - show vehicle compass
direction: Boolean - show tank direction indicator (not present in vanilla Arma 3)
menu: Boolean - show commanding menu (hides HC related menus)
group: Boolean - show group info bar (hides squad leader info bar)
cursors: Boolean - show HUD weapon cursors (connected with scripted HUD)